blueprints/ember-bootstrap/index.js

```diff
@@ -178,13 +180,24 @@ module.exports = {
     let file = 'ember-cli-build.js';
     let bootstrapVersion = this.bootstrapVersion;
     let preprocessor = this.preprocessor;
+    let config = this.retrieveExistingConfiguration();
     let settings = {
       bootstrapVersion
     };
 
-    settings.importBootstrapFont = (bootstrapVersion === 3);
+    if (bootstrapVersion === 4) {
+      settings.importBootstrapFont = false;
+    } else if (config.hasOwnProperty('importBootstrapFont')) {
+      settings.importBootstrapFont = config.importBootstrapFont;
+    } else {
+      settings.importBootstrapFont = true;
+    }
 
-    settings.importBootstrapCSS = (preprocessor === 'none');
+    if (config.hasOwnProperty('importBootstrapCSS')) {
```

Comment:
I debated that. The font setting is a strict requirement because the fonts aren't available in BS4. For the CSS, while it seems wasteful to import the CSS when you have a preprocessor, it's not wrong, so I chose to assume they did it for a reason rather than assume they were wrong.